SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -83,6 +106,32 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorCloudLauncherSettings : public UObje
 			: SnapshotPath.FilePath;
 	}
 
+	void SetPrimaryRegionCode(const ERegionCode::Type RegionCode);
+	FORCEINLINE FText GetPrimaryRegionCode() const
+	{
+		UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
+
+		if (!pEnum || !IsRegionCodeValid(PrimaryDeploymentRegionCode))
+		{
+			return FText::FromString(TEXT("Invalid"));
+		}
+
+		return pEnum->GetEnumTextByValue(static_cast<int64>(PrimaryDeploymentRegionCode.GetValue()));
+	}
+
+	void SetSimulatedPlayerRegionCode(const ERegionCode::Type RegionCode);
+	FORCEINLINE FText GetSimulatedPlayerRegionCode() const
+	{
+		UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
```

Comment:
This code for getting a string of an enum is repeated, could we make it into its own function and call it in the two getter functions?

Reply:
Makes sense - will refactor it 👍

Reply:
As discussed offline instead of extracting the functionality I removed the `pEnum == nullptr` check and left the two functions since they're quite short anyway.